src/main/java/org/zalando/nakadi/service/subscription/zk/ZkSubscriptionNode.java

```diff
@@ -26,15 +28,15 @@ public void setPartitions(final Partition[] partitions) {
         this.partitions = partitions;
     }
 
-    public void setSessions(final Session[] sessions) {
+    public void setSessions(final Collection<Session> sessions) {
         this.sessions = sessions;
     }
 
     public Partition[] getPartitions() {
         return partitions;
     }
 
-    public Session[] getSessions() {
+    public Collection<Session> getSessions() {
```

Comment:
the method is not used, removing it, you can avoid allocation empty array for sessions

Reply:
I can not, because there will be PR police who will ask me to put Nullable somewhere

Reply:
done